package.json

```diff
@@ -7,7 +7,7 @@
     "license": "Apache License Version 2.0",
     "publisher": "atsushieno",
     "engines": {
-        "vscode": "^1.1.2"
+        "vscode": "^1.25.0"
```

Comment:
I believe this is npm package version. The latest version for vscode package is 1.1.21. (I made similar mistake before)

Reply:
https://code.visualstudio.com/docs/extensions/publish-extension#_visual-studio-code-compatibility
It seems the doc says engines.vscode points on vscode end-user version.

Reply:
Ohh, indeed. Then this should be right. Thanks for the link.